src/decode.h

```diff
@@ -434,6 +434,9 @@ typedef struct Packet_
     uint16_t vlan_id[2];
     uint8_t vlan_idx;
 
+    uint16_t mpls_id[2];
```

Comment:
mpls label space seems to limited only by packet size (or mtu), so 2 might be a bit too low. I see limits being used in practice between 4 and 6 when I google it.

A second issue is that the label appears to be 20 bits in size, so a uint16_t is too small.